internal/models/feature.go

```diff
@@ -38,7 +38,7 @@ func (f Feature) FindBackground(astScenarioID string) *messages.Background {
 		}
 	}
 
-	return bg
+	return nil
```

Comment:
This change reverts https://github.com/cucumber/godog/commit/5414f3c5da2714be31ee25ff0ad8c53ee3d21816#diff-11cc1058f443cce780d83b56496302454770d91e64cd1f228d2d45d38598e9a4R41.

@mbow could you please check if that change was intended?